tests/data/patroni.service

```diff
@@ -0,0 +1,22 @@
+#########################################################################################
```

Comment:
Could this file not just be templated out at the time when the tests need it? Seems like it'll be a pain to keep this in sync with the template?